src/h3/_cy/latlng.pyx

```diff
@@ -245,3 +246,46 @@ cpdef double great_circle_distance(
         raise ValueError('Unknown unit: {}'.format(unit))
 
     return d
+
+# todo: one might want to move the following to a separate vertex.pyx file
```

Comment:
I'm supportive of moving this to a separate `vertex.pyx` file in this PR :)

Reply:
Yeah but I thought that would be a larger change of structure... I wrote this comment in case there would be more vertex-related functions in the future. IMHO the current structure is good enough, but I can move it to a separate file as well if it's very beneficial :-)

Reply:
I appreciate you asking! I think your suggestion to move the vertex code to its own file is a good one. :)

Reply:
Ok I'd do so.